packages/yum/test.sh

```diff
@@ -54,14 +54,7 @@ case ${package} in
     service_name=mariadb
     ha_mroonga_so=ha_mroonga_official.so
     test_package_name=MariaDB-test
-    case ${mysql_version} in
-      10.5|10.6|10.11)
-        baseurl=https://yum.mariadb.org/${mysql_version}/rhel${major_version}-amd64
-        ;;
-      *)
-        baseurl=https://yum.mariadb.org/${mysql_version}/centos${major_version}-amd64
-        ;;
-    esac
+    baseurl=https://yum.mariadb.org/${mysql_version}/rhel${major_version}-amd64
```

Comment:
```suggestion
    baseurl=https://yum.mariadb.org/${mysql_version}/rhel/${major_version}/amd64
```

https://mariadb.org/download/?t=repo-config&d=Red+Hat+Enterprise+Linux+8&v=11.4&r_m=yamagata-university で生成される感じだと今はこっちの方が推奨っぽい。

Reply:
そのパスだと `amd64` の代わりに `x86_64` っぽい。